src/LiquidityPool.sol

```diff
@@ -42,6 +43,7 @@ contract LiquidityPool is Initializable, OwnableUpgradeable, UUPSUpgradeable, IL
     uint128 public totalValueInLp;
 
     address public DEPRECATED_admin;
+    address public treasury;
```

Comment:
Adding new state variables in the upgradeable contract must be done in append-only manner only.
https://docs.openzeppelin.com/learn/upgrading-smart-contracts#upgrading

But in this case, I think we can re-purpose the deprecated variable `DEPRECATED_admin` for `treasury` :)

Reply:
I see this conversation is marked resolved, but Sykos point has not been addressed in this commit

Reply:
@vvalecha519